packages/@glimmerx/component/index.ts

```diff
@@ -1,3 +1,6 @@
 export { default } from '@glimmer/component';
 export { tracked } from '@glimmer/tracking';
-export { hbs } from './src/hbs';
+
+export function hbs(_strings: TemplateStringsArray) {
+  throw new Error('hbs template should have been compiled at build time');
+}
```

Comment:
Can we avoid duplicating this file? Specifically, deleting `packages/@glimmerx/component/index.ts` and adding a `main` entry point to `package.json` that points at `addon/index.ts`.

Reply:
I wanted to avoid pulling in `ember-cli-typescript`. I can see if it'll work with the standard TS compiler approach, but I don't think it's worth additional build complexity otherwise.

Reply:
Ya, seems fine for now. As things progress we can reevaluate...